File: server/server/Entities/Project.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;
using System.ComponentModel.DataAnnotations.Schema;

namespace server.Entities
{
	[Table("projects")]
    public class Project
    {
        [Key]
        public int id { get; set; }
        public String title { get; set; }
		public String subtitle { get; set; }
		public String description { get; set; }
		public String repo { get; set; }
		public String link { get; set; }
		public String image { get; set; }
    }
}

File: webapp/src/app/models/project.ts
class Project {
  title: string;
  subtitle: string;
  description: string;
  image: string; // TODO: base64 string?
  link: string;
  repo: string;

  public constructor(project: any) {
    this.title = project.title;
    this.subtitle = project.subtitle;
    this.description = project.description;
    this.image = project.image;
    this.link = project.link;
    this.repo = project.repo;
  }
}
